path/src/browser.ps1

```
@@ -0,0 +1,34 @@
+			& "$env:TEMP\$ChromeSetup" /install
+			$Process2Monitor = "ChromeSetup"
+			do {
+				Start-Sleep -Seconds 2
+			} while (Get-Process | Where-Object { $Process2Monitor -contains $_.Name } | Select-Object -ExpandProperty Name)
+			Remove-Item "$env:TEMP\$ChromeSetup" -ErrorAction SilentlyContinue
+
+			New-Item -Path "$FOUNT_DIR/data/installer" -ItemType Directory -Force | Out-Null
+			Set-Content "$FOUNT_DIR/data/installer/auto_installed_chrome" '1'
+			RefreshPath
+		}
+	} catch { <# ignore #> }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)